### ISSUE TITLE
env-injector should exit with error when it failed to get parameters

### ISSUE THREAD

**Issue by orangain** [okzk/env-injector#1]
I would like to discuss about design of error handling in env-injector. I believe env-injector should exit with error when it failed to get parameters rather than ignore the failure. There are following reasons:

1. Ignoring failure make it hard to notice the failure. 
2. Command execution with unexpected environment variables may cause unexpected result. This may be worse than error.

## Environment

env-injector: 0.0.2

## How to reproduce

Execute a command (e.g. env) using env-injector without permission of `ssm:GetParametersByPath`.

## Expected behavior

env-injector exits with error and the command is NOT executed.

## Actural behavior

env-injector ignores access-denied error silently and the command is executed without environment variables to be injected from Parameter Store. 

```
$ ENV_INJECTOR_VERBOSE=1 ENV_INJECTOR_PATH=/prod env-injector env
2017/10/14 04:26:44 parameter path: /prod
2017/10/14 04:26:44 no explicit region configuration. So now retrieving ec2metadata...
2017/10/14 04:26:44 failed to get by path: /prod
2017/10/14 04:26:44 AccessDeniedException: User: arn:aws:sts::xxxxxxxxxxxx:assumed-role/yyyy/i-nnnnnnn is not authorized to perform: ssm:GetParametersByPath on resource: arn:aws:ssm:ap-northeast-1:xxxxxxxxxxxx:parameter/prod
	status code: 400, request id: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
2017/10/14 04:26:44 no parameter prefix specified, skipping injection by prefix
ENV_INJECTOR_PATH=/prod
ENV_INJECTOR_VERBOSE=1
XDG_SESSION_ID=1
TERM=xterm-color
SHELL=/bin/bash
...
```

**Comment by okzk** [comment]
I agree to your proposal, and will release fixes in a few days.

Note: 
On the one hand, env-injector will keep ignoring permission errors when injecting grouped parameters without permission of `ssm:GetParameters`, because there may be empty environment variables other than for injection purposes.

**Comment by orangain** [comment]
Thanks! I'm looking forward the next release.

As you say, I think it reasonable to ignore permission errors for grouped parameters.

**Comment by okzk** [comment]
Just released [v0.0.3](https://github.com/okzk/env-injector/releases/tag/v0.0.3).

@orangain Please check it out.

**Comment by orangain** [comment]
Think you for the quick release! Now env-injector exits with error as expected.
